packages/jest-mock/src/index.ts

```diff
@@ -899,7 +911,7 @@ class ModuleMockerClass {
     return metadata;
   }
 
-  isMockFunction(fn: any): boolean {
+  isMockFunction<T, Y extends unknown[]>(fn: any): fn is Mock<T, Y> {
```

Comment:
I'm not sure if this makes sense or not

Reply:
Y is optional and `unknown[]` by default, so you can omit that. Docs say that _type gruard_ performs runtime check, but I don't see a change in the build output. Anyway, it makes sense to me

Reply:
the built output is the same, this is just us telling the type system that if the function returns `true` what the type is. Basically a `typeof`